specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -1052,6 +1052,56 @@
         }
       }
     },
+    "/routing/jobs/{jobId}/assignments/{assignmentId}:unassign": {
+      "post": {
```

Comment:
I am unfamiliar with the service so I am not sure whether un-assign would also mean delete. However, if it removes/deletes the 'assignment' then perhaps this should be a DELETE operation?